packages/@okta/vuepress-theme-prose/components/Footer.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <footer class="app-footer">
-    <p class="app-footer--need-support">
-      <span class="italic">Need support?</span> Ask on the
-      <SmartLink
-        :item="{ link: config.forum_url, target: '_self' }"
-        classes="link link--red"
-      >
-        forum.
-      </SmartLink>
-    </p>
     <div class="footer--columns">
+      <div class="footer--column need-support">
+        <h4 class="column--header">{{ config.footer_nav.need_support.heading }}</h4> 
+        <p>
+          You can reach us directly at developers@okta.com or you can also ask us on the 
```

Comment:
@albertchen-okta can you confirm if we want to be putting the email address back here or not?

Reply:
Maybe I'd better put it in config?